<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AES</class>
 <widget class="QWidget" name="AES">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#222f4d</string>
  </property>
  <widget class="QLabel" name="header">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>AES-Symmetric Ciphers</string>
   </property>
  </widget>
  <widget class="QLabel" name="mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modeOptions">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>751</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #f9f8fa;
    color: black;
    border-radius: 6px;
}

QComboBox QAbstractItemView {
    background-color: #f9f8fa;
    color: black;
    border-radius: 6px;
    selection-background-color: #aeb0b3; /* Change this color for the selected item in the dropdown list */
}

</string>
   </property>
   <item>
    <property name="text">
     <string>ECB (electronic code book)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CBC (cipher block chaining)</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="enc">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00a1ca;
color: black;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dec">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00a1ca;
color: black;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QLabel" name="output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="outputText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>851</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#f9f8fa;
color:balck;
border-radius: 6px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>650</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00a1ca;
color: black;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Init. vector</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="initVector">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#f9f8fa;
color:balck;
border-radius: 6px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>89</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#f9f8fa;
color:balck;
border-radius: 6px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="key">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#f9f8fa;
color:balck;
border-radius: 6px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00a1ca;
color: black;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
